app/app/management/commands/sync_orgs_repos.py

```diff
@@ -0,0 +1,189 @@
+                        profile.repos.remove(y)
+
+                    lsynced.append(handle)
+                else:
+                    return lsynced
+
+                members_to_sync = []
+                if profile.organizations is None:
+                    print("no organizations to sync")
+                    return []
+
+                for org in profile.organizations:
+                    try:
+                        if org in orgs_synced:
+                            print(f'{org} has been synced already')
+                            continue
+
+                        orgs_synced.append(org)
+                        db_org = Organization.objects.get_or_create(name=org)[0]
+
+                        print(f'Syncing Organization: {db_org.name}')
+                        profile.profile_organizations.add(db_org)
+                        org_members = get_organization(
+                            db_org.name,
+                            '/members',
+                            (handle, access_token)
+                        )
+
+                        if 'message' in org_members:
+                            print(org_members['message'])
+                            continue
+
+                        for member in org_members:
+
+                            try:
+                                membership = get_organization(
+                                    db_org.name,
+                                    f'/memberships/{member["login"]}',
+                                    (handle, access_token)
+                                )
+                                if 'message' in membership:
+                                    print(membership['message'])
+                                    continue
+                                role = membership['role'] if 'role' in membership else "member"
+                                db_group = Group.objects.get_or_create(name=f'{db_org.name}-role-{role}')[0]
+                                db_org.groups.add(db_group)
+                                member_profile_obj = Profile.objects.get(
+                                    handle=member['login'],
+                                    user__is_active=True
+                                )
+                                member_profile_obj.user.groups.add(db_group)
+                                members_to_sync.append(member['login'])
+                            except Exception as e:
+                                # print(f'An exception happened in the Organization Loop: handle {member["login"]} {e}')
+                                continue
+
+                        org_repos = get_organization(
+                            db_org.name,
+                            '/repos',
+                            (handle, access_token)
+                        )
+
+                        if 'message' in org_repos:
+                            print(org_repos['message'])
+                            continue
```

Comment:
^ remove the if block ?

Reply:
no, same as above, anytime we see a message key in the returning object, we want to log it and skip to the next element, the error's are vague but are only caused by a couple of things, permissions errors are the most prevelant